Source/glTFRuntime/Public/glTFDataAsset.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015-2022 PixelPool
+
+#pragma once
+
+#include "Engine/DataAsset.h"
+#include "glTFDataAsset.generated.h"
+
+UCLASS()
+class GLTFRUNTIME_API UglTFDataAsset
+	: public UDataAsset
+{
+	GENERATED_BODY()
+
+public:
+
+	UMaterialInterface* GetOpaqueMaterial() const { return OpaqueMaterial; };
```

Comment:
Not sure if this is a problem at the moment, but if you directly return the `UMaterialInterface*` from the DataAsset, if any changes are made, they might also change the material in the DataAsset.

Maybe it would be safer to return something like this:
```cpp
UMaterialInterface* GetOpaqueMaterial() const { return UMaterialInstanceDynamic::Create(OpaqueMaterial, nullptr); }
```

Reply:
Discussed with Yannick, we don't want to change to much in the plugin and this is kind of how it worked before as well. And didn't see any issues during QA when switching between different glbs